src/gluonts/dataset/stat.py

```diff
@@ -39,22 +40,18 @@ class ScaleHistogram:
     empty_target_count
     """
 
-    @validated()
-    def __init__(
-        self,
-        base: float = 2.0,
-        bin_counts: Optional[dict] = None,
-        empty_target_count: int = 0,
-    ) -> None:
-        self._base = base
+    _base: float = 2.0
+    bin_counts: Optional[dict] = None
+    empty_target_count: int = 0
+
+    def __post_init_post_parse__(self):
         self.bin_counts = defaultdict(
-            int, {} if bin_counts is None else bin_counts
+            int, {} if self.bin_counts is None else self.bin_counts
         )
-        self.empty_target_count = empty_target_count
-        self.__init_args__ = dict(
-            base=self._base,
+        self.__init_kwargs__ = dict(
```

Comment:
If the value is mutable, we should set the `__init_kwargs__` in `__post_init_post_parse__` and use `self.xxx`